docs/DatabaseCatalogAPI.yaml

```diff
@@ -643,29 +635,51 @@ components:
     provider:
       name: provider
       in: query
-      description: Filter feeds by provider name using the LIKE operation. Phrase is matched case insensitively to actual provider names.
+      description: List only feeds with the specified value. Can be a partial match. Case insensitive.
       required: false
       schema:
         type: string
-        example: London Transit
+        example: Los Angeles Department of Transportation
```

Comment:
Would be nice to return an example for /feeds that shows multiple feeds rather than just one. How about Intercity Transit, with the producer_url of https://its.rideralerts.com?

Reply:
If I just filter with `Intercity Transit` it does return 2 feeds. But if I add the producer_url then it returns only one feed.
We want examples for each parameter, right? 
So that means we would need a query that returns 2 or more feeds, but that would have the same provider, producer_url, country_code, subdivision_name, municipality, latitudes, longitudes. 
Any parameter that we would want different would mean no example for that parameter.
Does this make sense?

Reply:
True, that does make sense. We can leave the example as is then.